src/pages/PassportRedemption/TicketRow.tsx

```diff
@@ -51,15 +51,30 @@ const TicketRow = (props: Props) => {
       case RowStatuses.Active:
         return `READY TO REDEEM`;
       default:
-        return `${3 - props.stamps.length} MORE STAMPS UNTIL YOUR NEXT REWARD`;
+        if (props.stamps.length === 0) return;
+        return `${3 - props.stamps.length} MORE ${
+          3 - props.stamps.length > 1 ? 'STAMPS' : 'STAMP'
```

Comment:
I wonder if we could add a util that pluralizes words — I've seen this ternary littered about our code

Reply:
i could do this after we release passport v1